renaud.mk

```diff
@@ -1,4 +1,4 @@
 INSTANCE_ID = renaud
 VARS_FILE = vars_renaud.yaml
-
+PRINT_OUTPUT = /tmp
```

Comment:
yes, but they don't need to be in git...
I'd like to prevent that everyone sets a certain variable to the same value in his make file that's why I suggested to put it in geoportailv3.mk